src/facade/dragonfly_listener.h

```diff
@@ -35,12 +35,16 @@ class Listener : public util::ListenerInterface {
   bool AwaitDispatches(absl::Duration timeout,
                        const std::function<bool(util::Connection*)>& filter);
 
+  bool IsAdminInterface() const;
+  void SetAdminInterface(bool is_admin = true);
```

Comment:
We already have Connection::IsAdmin(), could we maybe consolidate them to use the same logic? (either inspect the port or explicitly set like here)

Reply:
Oooo, I did NOT expect the implementation of `Connection::IsAdmin` to be what it is :laughing:
Changed it to query the Listener.